clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -485,6 +485,12 @@ static void InitializeStandardPredefinedMacros(const TargetInfo &TI,
 
     if (LangOpts.SYCLValueFitInMaxInt)
       Builder.defineMacro("__SYCL_ID_QUERIES_FIT_IN_INT__", "1");
+
+    // Set __SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ macro for
+    // both host and device compilations if -fintelfpga flag is present.
+    if (LangOpts.AheadOfTimeFPGACompilation) {
+      Builder.defineMacro("__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__");
+    }
```

Comment:
```suggestion
    if (LangOpts.AheadOfTimeFPGACompilation)
      Builder.defineMacro("__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__");
```
Drops the braces as part of the usual community coding guidelines.